dali/pipeline/data/types.h

```diff
@@ -264,6 +264,21 @@ class DLL_PUBLIC TypeInfo {
   DLL_PUBLIC void Copy(void *dst, const void **srcs, const Index *sizes, int n, cudaStream_t stream,
                        bool use_copy_kernel = false) const;
 
+  /**
+   * @brief Copies from SrcBackend contiguous buffer to DstBackend scattered locations
+   * @param dsts destination pointers
+   * @param src source pointer
+   * @param sizes number of elements for each of the pointers specified in dsts
```

Comment:
`number of elements` or `number of bytes`?

Reply:
it's number of elements